tools/fedora-test.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-[[ -d ${0%/*} ]] && cd "${0%/*}"
+[[ -d ${0%../*} ]] && cd "${0%../*}"
```

Comment:
`${0%/*}` is now the `tools` directory. We want `tools/..` to get to the dracut source root directory.
```suggestion
[[ -d ${0%/*} ]] && cd "${0%/*}"/../
```

Reply:
Done